platform/android/detect.py

```diff
@@ -176,19 +185,35 @@ def configure(env: "SConsEnvironment"):
         )
     )
 
+    has_swappy = detect_swappy()
+    if not has_swappy:
+        print_warning(
+            "Swappy Frame Pacing not detected! It is strongly recommended you download it from https://github.com/darksylinc/godot-swappy/releases and extract it so that thirdparty/swappy-frame-pacing/arm64-v8a/libswappy_static.a can be found.\n"
```

Comment:
> thirdparty/swappy-frame-pacing/arm64-v8a/libswappy_static.a

This should probably be generic since the architecture directory will change based on the arch. For example:
`thirdparty/swappy-frame-pacing/<arch_dir>/libswappy_static.a`

Reply:
This is more a social thing, which I'm not fully satisfied with the current wording (nor with all the alternatives I can think of).

Here's the rationale let's assume a very, very dumb user; I always assume worst case and then walk to the second-best case which is a smart user (best case is user starts reading the source code and figures it out, but that's an outlier):

**It says:** `thirdparty/swappy-frame-pacing/<arch_dir>/libswappy_static.a`
- User may reply "there is no <arch_dir> folder!!! What do I do???"
- Smart people will not be certain what folders are supposed to either. Just cross fingers they uncompressed it correctly.

**It says:** `thirdparty/swappy-frame-pacing/arm64-v8a/libswappy_static.a`
- User may reply "just arm64-v8a folder? But I am compiling for x86..."
- Smart people will figure out at least arm64-v8a is correct, and see the other folders and cross fingers they have all the platforms included.


**It says:** `thirdparty/swappy-frame-pacing/arm64-v8a/libswappy_static.a` + `thirdparty/swappy-frame-pacing/x86/libswappy_static.a` + (basically list all platforms)
- User will not read it. It was too long.
- Smart people will have no confusion.

There's basically no solution that will satisfy all cases. I think I prefer to list all platforms rather than use `<arch_dir>`. The goal is that when something goes wrong, we give enough information for users in hopes of **minimizing the amount people that will reach to tech support** (e.g. leave a ticket here, post a forum post in community channels, etc). And if they do reach support and end up being asked to upload the error message, the smart people will figure out what the user needs to do.

Reply:
Pushed new version with all 4 variations.